pkgs/tools/graphics/vulkan-helper/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, vulkan-loader
+, addOpenGLRunpath
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vulkan-helper";
+  version = "2023-09-16";
```

Comment:
`unstable-2023-09-16`

Reply:
Is "unstable" the right term, since upstream doesn't do releases at all?

Reply:
It's the convention. /shrug

Reply:
Done.